pkg/backend/display/progress.go

```diff
@@ -1004,6 +1116,7 @@ func (display *ProgressDisplay) ensureHeaderAndStackRows() {
 		display:              display,
 		tick:                 display.currentTick,
 		diagInfo:             &DiagInfo{},
+		policyInfo:           &PolicyInfo{},
```

Comment:
looks like this is always non-nil.  1. i don't know why we even need the wrapper around the array.  2. We def don't need nil checks for it :)